src/Symfony/Component/Finder/Finder.php

```diff
@@ -131,18 +134,21 @@ public function depth($level)
      *   $finder->date('until 2 days ago');
      *   $finder->date('> now - 2 hours');
      *   $finder->date('>= 2005-10-15');
+     *   $finder->date(['>= 2005-10-15', '>= 2005-05-27']);
```

Comment:
I suppose the first one should probably be `<=` for the range to make sense.

Reply:
Yes, it’s fixed.
I checked that i did not make the mistake in the tests and documentation.